packages/discord.js/package.json

```diff
@@ -72,7 +72,7 @@
     "@discordjs/util": "workspace:^",
     "@discordjs/ws": "1.1.1",
     "@sapphire/snowflake": "3.5.3",
-    "discord-api-types": "0.37.97",
```

Comment:
-types bumps should happen across all packages, not just djs

Ideally in a separate PR

Reply:
#10488 will bump types, then I'll rebase